LiskHQ/lisk-hub
# Upgrade vulnerable dependencies with no breaking changes - Closes #1210

### What was the problem?
- #1210

### How did I fix it?

### How to test it?

- install npm 6
- remove old packages ```rm -rf ./node_modules```
- run ```npm install``` and verify there are no critical dependencies on the output logs

### Review checklist
- The PR closes #1210, closes #1131
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
